### DISCUSSION TITLE
/bin/bash: line 234: /usr/local/bin/brew: No such file or directory

### DISCUSSION THREAD

**Discussion by rico345100** [Homebrew/discussions#4555] [Getting started]
### Output of `brew config`

```shell
zsh: command not found: brew
```

### Output of `brew doctor`

```shell
zsh: command not found: brew
```

### Description of issue

Typed this in my brand new macbook air m1
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"

Got this
mingyuchoi@MinGyuui-MacBookAir ~ % /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
==> Checking for `sudo` access (which may request your password)...
Password:
==> This script will install:
/usr/local/bin/brew
/usr/local/share/doc/homebrew
/usr/local/share/man/man1/brew.1
/usr/local/share/zsh/site-functions/_brew
/usr/local/etc/bash_completion.d/brew
/usr/local/Homebrew

Press RETURN/ENTER to continue or any other key to abort:
==> /usr/bin/sudo /usr/sbin/chown -R mingyuchoi:admin /usr/local/Homebrew
==> Downloading and installing Homebrew...
HEAD is now at e9a1cd41d Merge pull request #15510 from ZhongRuoyu/actionlint-macos-13
/bin/bash: line 234: /usr/local/bin/brew: No such file or directory
Failed during: /usr/local/bin/brew update --force --quiet

HOW TO FIX THIS

**Comment by rico345100** [comment]
HOW TO FIX THIS!

**Comment by osalbahr** [comment]
Try running the following command:
```
file /usr/local
```
Does it fail? If so, are you able to manually create that directory and rerun `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`?
